src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Events/EventPage.razor

```diff
@@ -35,7 +35,7 @@
         var entity = await _behind.Read(default);
         if (entity == null)
         {
-           GuardHelper.ThrowIfNull(entity, "Event cannot be null here");    
+           GuardHelper.ThrowIfNull(entity);    
         }
```

Comment:
Haha, not quite. Take a careful look at lines I've marked here and then look at the definition of `ThrowIfNull` again. If you still can't see it - replace the invocation of `ThrowIfNull`  with it's definition and you'll see what I mean.